### PR TITLE
Add support for uppercase image extension

### PR DESCRIPTION
**Pull request by tsvietOK**
Like .JPG, .PNG etc.
Fixed a crash when you right clicked a file without extension.

### CODE REVIEW

File: Files/View Models/InteractionViewModel.cs [hunk 1]
```diff
@@ -53,7 +53,10 @@ public void CheckForImage()
             {
                 string ItemExtension = (App.CurrentInstance.ContentPage as BaseLayout).SelectedItem.FileExtension;
 
-                if (ItemExtension == ".png" || ItemExtension == ".jpg" || ItemExtension == ".bmp" || ItemExtension == ".jpeg")
+                if (ItemExtension.ToLower() == ".png"
```
Comment (yaichenbaum) [inline Files/View Models/InteractionViewModel.cs:56]: When I tried doing this the application would crash when right clicking on an image file, have you confirmed that this works?

Comment (tsvietOK) [reply Files/View Models/InteractionViewModel.cs]: It is string, ToLower() can't crash app. From my side it is working fine.

Comment (duke7553) [reply Files/View Models/InteractionViewModel.cs]: I would prefer using ItemExtension.Equals(".png", StringComparison.OrdinalIgnoreCase) for this one.

Comment (duke7553) [reply Files/View Models/InteractionViewModel.cs]: Just to keep things consistent.

Comment (tsvietOK) [reply Files/View Models/InteractionViewModel.cs]: Done

Comment (yaichenbaum) [reply Files/View Models/InteractionViewModel.cs]: @tsvietOK Can you test that the functionality still works after doing the changes @duke7553 requested?

Comment (tsvietOK) [reply Files/View Models/InteractionViewModel.cs]: Yeah, i will test it little bit later

Comment (tsvietOK) [reply Files/View Models/InteractionViewModel.cs]: @yair100 It is working

**Pull request merged**